### ISSUE TITLE
phx-keyup overrides phx-keydown

### ISSUE THREAD

**Issue by skylerparr** [phoenixframework/phoenix_live_view#94]
phx-keyup appears to override phx-keydown when used together.

### Environment

* Elixir version (elixir -v):
```
Erlang/OTP 21 [erts-10.2.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

Elixir 1.7.4 (compiled with Erlang/OTP 20)
```
* Phoenix version (mix deps):
```
* phoenix 1.4.2 (Hex package) (mix)
  locked at 1.4.2 (phoenix) 3a1250f2
  ok
* phoenix_live_reload 1.2.0 (Hex package) (mix)
  locked at 1.2.0 (phoenix_live_reload) 3bb31a9f
  ok
* phoenix_html 2.13.1 (https://github.com/phoenixframework/phoenix_html.git) (mix)
  locked at fd5036c
  ok
* phoenix_live_view 0.1.0 (https://github.com/phoenixframework/phoenix_live_view.git) (mix)
  locked at 56482db
  ok
```
* NodeJS version (node -v):
```
v8.15.0
```
* NPM version (npm -v):
```
6.4.1
```
* Operating system:
```
Linux MintyMac 4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
### Repro code
```
defmodule DemoWeb.TestLive do
  use Phoenix.LiveView

  def mount(_session, socket) do
    {:ok, socket}
  end

  def render(assigns) do
    ~L"""
    <div phx-keydown="keydown" phx-keyup="keyup" phx-target="window">
    </div>
    """
  end

  def handle_event("keyup", key, socket) do
    IO.inspect key
    {:noreply, socket}
  end

  def handle_event("keydown", key, socket) do
    IO.inspect key
    {:noreply, socket}
  end

end
```

### Actual behavior
When pressing a key I get the output of the key pressed only on the key up event. When removing the `phx-keyup` attribute, the keydown fires as expected. 

### Expected behavior
Should be getting two separate events for `key up` and `key down`, instead of only the `key up` event.

### Notes
Might be related to https://github.com/phoenixframework/phoenix_live_view/issues/83

**Comment by chrismccord** [comment]
Closed via #99

**Comment by skylerparr** [comment]
Impressive turn around. Thanks Chris!
